osu.Game.Rulesets.Soyokaze/Mods/SoyokazeModHidden.cs

```diff
@@ -13,6 +15,19 @@ namespace osu.Game.Rulesets.Soyokaze.Mods
 {
     public class SoyokazeModHidden : ModHidden
     {
+        [SettingSource("Fading Approach Circle", "Allow approach circles to fade instead of disappearing")]
+        public Bindable<bool> FadingApproachCircle { get; } = new BindableBool(false);
```

Comment:
Honestly, I really like this setting. I think this setting should be `true` by default. Like let's be real, hidden is actually unplayable without this setting.

Reply:
If this value is ``true`` by default, will ``Description`` also be changed? I think it should be something like "Play with fading circles"

Reply:
True, HD is actually pretty readable with this, so the old desc doesn't make much sense.